### PR TITLE
Chore: Update changelog

### PR DESCRIPTION
**Pull request by aknuds1**
**What this PR does / why we need it**:
Update the changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -55,6 +55,17 @@ With this feature, you can specify the local time zone of the service or system
 
 We have also extended the time zone options so you can select any of the standard [ISO 8601 time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
 
+# 7.0.0-beta3 (2020-05-08)
+
+### Features / Enhancements
+* **Docker**: Upgrade to Alpine 3.11. [#24056](https://github.com/grafana/grafana/pull/24056), [@aknuds1](https://github.com/aknuds1)
+* **Forms**: Remove Forms namespace [BREAKING]. [#24378](https://github.com/grafana/grafana/pull/24378), [@tskarhed](https://github.com/tskarhed)
```
Comment (dprokop) [inline CHANGELOG.md:62]: @tskarhed can you provide some details here about the  possible consequences of this change?

**Review by dprokop — APPROVED** [review]

**Review by marefr — APPROVED** [review]
LGTM

Maybe copy breaking changes from beta2 (together with Tobias changes).

File: CHANGELOG.md (hunk 1 above)
Comment (tskarhed) [inline CHANGELOG.md:62]:
```suggestion
* **Forms**: Remove Forms namespace [BREAKING]. Will cause all `Forms` imports to stop working. See migration guide in [@grafana/ui changelog](https://github.com/grafana/grafana/blob/master/packages/grafana-ui/CHANGELOG.md)[#24378](https://github.com/grafana/grafana/pull/24378), [@tskarhed](https://github.com/tskarhed)
```

File: packages/grafana-ui/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# 7.0.0-beta3 (2020-05-08)
+
+### Features / Enhancements
+* **Forms**: Remove Forms namespace [BREAKING]. [#24378](https://github.com/grafana/grafana/pull/24378), [@tskarhed](https://github.com/tskarhed)
```
Comment (tskarhed) [inline packages/grafana-ui/CHANGELOG.md:4]:
```suggestion
* **Forms**: Remove Forms namespace [BREAKING]. Will cause all `Forms` imports to stop working. See migration guide below. [#24378](https://github.com/grafana/grafana/pull/24378), [@tskarhed](https://github.com/tskarhed)
```

**Review by tskarhed — APPROVED** [review]

**Pull request merged**